package.json

```diff
@@ -20,8 +20,10 @@
   "license": "MIT",
   "dependencies": {
     "@kalkih/lz-string": "^1.4.5",
-    "lit-element": "^2.2.1",
```

Comment:
Why is there an update in `package.json`?
I don't think your PR requires any update there.
I'd rebase from dev and reset this file to it's current state.